core/logger/Logger.h

```diff
@@ -21,6 +21,7 @@
 #include <spdlog/spdlog.h>
 
 namespace logtail {
+    using namespace spdlog::level;
```

Comment:
using namespace in .h is not allowed. please use complete namespace before types